x/metoken/README.md

```diff
@@ -0,0 +1,326 @@
+# MeToken Module
+
+## Abstract
+
+This document specifies the `x/metoken` module of the Umee chain.
+
+The `metoken` module allows users to swap and redeem accepted assets for an Index meToken. The Index meToken will
+maintain the parity between underlying assets given a specific configuration.
+
+The `metoken` module depends directly on `x/leverage` for supplying and withdrawing assets, and the cosmos `x/bank`
+module as these all affect account balances.
```

Comment:
80% of assets that are swapped for meTokens will be deposited on Umee through the leverage module. We should add a section to the spec regarding this interaction

Reply:
I see it in swapping